portfolio/src/main/webapp/repeatedElements.js

```diff
@@ -145,7 +145,7 @@ function generateTiles(imgPrefix, tileIndexs, elId) {
  * Generate a flexbox for every set of tiles
  */
 function generateAllTiles(prefix, tilesJSON) {
```

Comment:
don't use all caps in variable names unless they are constants.

Reply:
noted